docs/concepts/overview/extending.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Extending your Kubernetes Cluster
+approvers:
+- erictune
+- lavalamp
+- cheftako
+- chenopis
+---
+
+{% capture overview %}
+
+Kubernetes has many ways to configure and extend it.  As a result,
```

Comment:
Suggest:

"Kubernetes is highly configurable and extensible.  As a result,"

Reply:
Thanks, accepted suggestion.